src/main/java/net/mcreator/ui/modgui/BiomeGUI.java

```diff
@@ -62,12 +62,15 @@
 	private final JSpinner treesPerChunk = new JSpinner(new SpinnerNumberModel(3, 0, 256, 1));
 	private final JSpinner grassPerChunk = new JSpinner(new SpinnerNumberModel(4, 0, 256, 1));
 	private final JSpinner seagrassPerChunk = new JSpinner(new SpinnerNumberModel(20, 0, 256, 1));
+	private final JSpinner bamboosPerChunk = new JSpinner(new SpinnerNumberModel(0, 0, 256, 1));
 	private final JSpinner flowersPerChunk = new JSpinner(new SpinnerNumberModel(4, 0, 256, 1));
+	private final JSpinner doubleFlowersPerChunk = new JSpinner(new SpinnerNumberModel(4, 0, 256, 1));

```

Comment:
If the default value of this is 4, set the default value to 4 in Biome.java too, so existing mod elements get this default value added.